### PR TITLE
Added DisneyPlus Clone

### PR DESCRIPTION
**Pull request by Mannxxx**
I've added disneyplus clone keeping in mind the alphabetical order. The link is working properly and the quality is also amazing. Worth adding to the list.

### CODE REVIEW

**Comment by crazy-man22** [comment]
> I've added disneyplus clone keeping in mind the alphabetical order. The link is working properly and the quality is also amazing. Worth adding to the list.

the live demo link is not working

**Comment by Mannxxx** [comment]
@crazy-man22. The issue is from the db and hosting but it should work fine in your project

**Comment by crazy-man22** [comment]
> @crazy-man22. The issue is from the db and hosting but it should work fine in your project

if it's your project, you can deploy it on Vercel.
The README has to have a correct demo link so that vistors can try it online easily

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ _(scroll right on table to see all 5 columns)_
...
 | Disney+                        | [github.com (screenshot)](https://github.com/calebnance/expo-disneyplus/blob/master/.gh-assets/screenshare-4.png?raw=true)                                             | [GitHub](https://github.com/calebnance/expo-disneyplus)                                                                                    | React Native, expo                                                             | ![GitHub Repo stars](https://flat.badgen.net/github/stars/calebnance/expo-disneyplus)                                                                                                            |
+| Disneyplus                        | [github.com (screenshot)](https://disney-clone-d1e27.web.app/)                                             | [GitHub](https://github.com/CleverProgrammers/cp-disney-plus-clone)                                                                                    | Javascript, ReactJS, expo                                                             | ![GitHub Repo stars](https://flat.badgen.net/github/stars/CleverProgrammers/cp-disney-plus-clone)   |
```
Comment (Pritamgit2002) [inline README.md:88]: Demo Link is not working !!!!

**Review by Pritamgit2002 — CHANGES_REQUESTED** [review]

**Comment by Pritamgit2002** [comment]
provide demo link

**Pull request closed without merging**